src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,46 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    flex-direction: row;
+    display: flex;
+    width: 16px;
+    height: 16px;
+    flex-shrink: 0;
+    align-items: center;
+    margin-right: 4px;
+    background: url("/images/star.svg") no-repeat  center center;
+    z-index: 1;
+  }
+
+  &--1 {
+    .stars__star:nth-child(-n+1) {
+      @include activeStar
+    }
+  }
+
+  &--2 {
```

Comment:
good job in making this style reusable, however i think, it's better to make use of [sass @for loop](https://sass-lang.com/documentation/at-rules/control/for/)